src/Slices/ServiceInventory/UI/Presenters/InstanceExpertActionPresenter.ts

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement } from "react";
+import { ServiceModel } from "@/Core";
+import { ActionPresenter, ServiceInstanceForAction } from "@/UI/Presenters";
+import { ExpertActions } from "../Components/ExpertActions";
+
+export class InstanceExpertActionPresenter implements ActionPresenter {
```

Comment:
Could you add explanation as to why we need 'presenters'

Reply:
I will add what it does, but reasoning behind whole concept is kind of unclear to me, but I didn't went to seek answer for that through whole TableProvider. I can try to look for that if you want :)